src/Blorc.OpenIdConnect/Models/Profile.cs

```diff
@@ -1,10 +1,45 @@
 ﻿namespace Blorc.OpenIdConnect
 {
+    using System.Collections.Generic;
     using System.Security.Claims;
+    using System.Text.Json;
     using System.Text.Json.Serialization;
 
     public class Profile
     {
+        [ClaimType("aud")]
+        [JsonPropertyName("aud")]
+        public JsonElement Aud
+        {
+            get;
+            set;
+        } = JsonDocument.Parse("[]").RootElement;
+
+        public string[] Audiences
+        {
+            get
+            {
+                if (Aud.ValueKind == JsonValueKind.Array)
+                {
+                    var result = new List<string>();
+                    using var enumerator = Aud.EnumerateArray();
+
+                    foreach (var item in enumerator)
+                    {
+                        result.Add(item.GetString()!);
+                    }
+
+                    return result.ToArray();
+                } 
+                else if (Aud.ValueKind == JsonValueKind.String)
+                {
+                    return [ Aud.GetString()! ];
```

Comment:
same here, do we want to support empty strings or return an empty array in case of empty string?